tools/analysis/analysis_runner.py

```diff
@@ -0,0 +1,77 @@
+# Analysis runner.
+import argparse
+import os
+import subprocess
+
+# TODO: Support customizing these.
+_TOPIC = 'projects/ossf-malware-analysis/topics/workers'
+_PACKAGES_BUCKET = 'ossf-malware-analysis-packages'
```

Comment:
These could be read from the same ENV vars as the other commands.

Reply:
Sure, done!